apps/meteor/app/cloud/server/functions/supportedVersionsToken/supportedVersionsToken.ts

```diff
@@ -10,6 +10,12 @@ import { settings } from '../../../../settings/server';
 import { generateWorkspaceBearerHttpHeader } from '../getWorkspaceAccessToken';
 import { supportedVersionsChooseLatest } from './supportedVersionsChooseLatest';
 
+declare module '@rocket.chat/license' {
+	interface ILicenseV3 {
```

Comment:
`question` Why this augmentation is needed?

Reply:
so far we dont have decided if the supported version is going to be added to the license, also I'm still considering where the types should live: cloud/license/supportedVersions